.circleci/config.yml

```
@@ -94,7 +94,11 @@ commands:
     steps:
       - restore_cache:
           key: 1-gems-{{ checksum "Gemfile.lock" }}
-      - run: bundle check || bundle install --path vendor/bundle --clean
+      - run: 
```

```suggestion
      - run:
```
There is a whitepsace here